### DISCUSSION TITLE
Anonymous Statistics

### DISCUSSION THREAD

**Discussion by reddog335** [rancher-sandbox/rancher-desktop#2671] [Q&A] [answered]
Hi all!  What endpoint(s) are the anonymous statistics sent and are they transferred over HTTPS?

**Comment by mook-as** [comment, marked as answer]
Hi! Currently the checkbox doesn't do anything, and there's nothing sent anywhere (but once it does get implemented, it would likely be over HTTPS, yes.) It just exists so that people have plenty of time to opt-out before it gets implemented (…eventually, probably).

There's a separate version check checkbox that contacts `https://desktop.version.rancher.io/`, as configured [here](https://github.com/rancher-sandbox/rancher-desktop/blob/v1.5.0/electron-builder.yml#L44).

**Reply by reddog335** [reply]
@mook-as  Thank you for answering my question, exactly what I was looking for....much appreciated!
